cmake/pcre2-config.cmake.in

```diff
@@ -126,6 +126,7 @@ if (PCRE2_FOUND)
     endif ()
     set_target_properties(PCRE2::${component} PROPERTIES
       IMPORTED_LOCATION "${PCRE2_${component}_LIBRARY}"
+      IMPORTED_IMPLIB "${PCRE2_${component}_LIBRARY}"
```

Comment:
this seems to be a somehow recent property, can you confirm it works in the required cmake version, and if not, whch version should be required instead?

Reply:
It's listed in https://cmake.org/cmake/help/v3.5/manual/cmake-properties.7.html#target-properties so it should be ok

Reply:
Yes, IMPORTED_IMPLIB is not a new thing.